src/main/java/de/retest/recheck/review/ignore/AttributeFilter.java

```diff
@@ -73,7 +73,7 @@ public String save( final Filter ignore ) {
 
 		@Override
 		protected Optional<Filter> load( final MatchResult regex ) {
-			return ChainableFilterLoaderUtil.load( regex, match -> loadSimpleAttributeFilter( match ) );
+			return ChainableFilterLoaderUtil.load( regex, this::loadSimple );
```

Comment:
I made a typo in my suggestion, which is why the build is failing:

```suggestion
			return ChainableFilterLoaderUtil.load( regex, this::loadSimpleAttributeFilter );
```

Reply:
Haha, soviel dazu wenn man das auf GitHub approved:
```
[ERROR] /home/travis/build/retest/recheck/src/main/java/de/retest/recheck/review/ignore/AttributeFilter.java:[76,71] invalid method reference
  cannot find symbol
    symbol:   method loadSimple()
    location: class de.retest.recheck.review.ignore.AttributeFilter.AttributeFilterLoader
```